src/main/java/seedu/address/ui/EventListPanel.java

```diff
@@ -14,7 +14,13 @@
  * Panel containing the list of items.
  */
 public class EventListPanel extends UiPart<Region> {
+
+    private static final int NUM_OF_ITEMS_TO_SCROLL = 5;
```

Comment:
I think there should be a comment over here to explain why the number 5 is chosen.

Reply:
Okay!